buildhub/urls.py

```diff
@@ -26,8 +26,6 @@ def serve(request, **kwargs):
     _, ext = os.path.splitext(request.path_info)
     if ext:
         return http.HttpResponseNotFound(request.path_info)
-    document_root = kwargs["document_root"]
-    assert os.path.isdir(document_root), document_root
```

Comment:
This causes problems in the dev environment where `/app/ui/build` exists in the image, but `/app` is stomped on by the contents of the repo so then `/app/ui/build` doesn't exist. In a dev environment, you run a `ui` service at port 3000 which handles the static assets, so this check just causes problems. I figured I'd just nix it.